### PR TITLE
[ENSWEB-6093] missing reference species and other bug fixes

### PR DESCRIPTION
**Pull request by jyothishnt**
- Decided to show reference species
- Fixed bug on filtering using menus

https://www.ebi.ac.uk/panda/jira/browse/ENSWEB-6093

http://ves-hx-78.ebi.ac.uk:9000/info/about/speciestree.html

### CODE REVIEW

**Comment by ens-ma7** [comment]
@jyothishnt sandbox URL please.

File: widgets/htdocs/widgets/95_SpeciesTree.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ Ensembl.SpeciesTree.displayTree = function(json, panel) {
   var theme =  Ensembl.SpeciesTree.tnt_theme_tree_simple_species_tree(json);
   theme(tnt.tree(), document.getElementById("species_tree"));
 },
-
+Ensembl._species_tree_json;
```
Comment (ens-ma7) [inline widgets/htdocs/widgets/95_SpeciesTree.js:18]: I cant see where this is used again. it looks like we dont need this variable or am i missing something

**Review by ens-ma7 — APPROVED** [review]

**Comment by ens-ma7** [comment]
Code change looks good and i tested it on the sandbox as well, looks ok. @jyothishnt probably best to get compara to test it before pushing the changes.

**Comment by jyothishnt** [comment]
> Code change looks good and i tested it on the sandbox as well, looks ok. @jyothishnt probably best to get compara to test it before pushing the changes.

Thanks @ens-ma7, compara have already tested 👍

**Pull request merged**